src/components/routing/routeProtectors/LoginGuard.js

```diff
@@ -12,7 +12,7 @@ export const LoginGuard = () => {
     return <Outlet />;
```

Comment:
@B1s9l make api login request. i could just set an arbitrary token value and then i am logged in, which is bad.

maybe we need to think about an `/auth` endpoint to check authorization of local auth token